datalad/distribution/tests/test_install.py

```diff
@@ -36,6 +37,11 @@
     InsufficientArgumentsError,
 )
 from datalad.support.gitrepo import GitRepo
+from datalad.support.network import get_local_file_url
+from ...tests.utils_testdatasets import (
```

Comment:
again relative versus absolute imports here

Reply:
even I would agree to use abs imports here. Design doc has it -- use relative based on "semantic" binding of the code

Reply:
Done